### PR TITLE
Handling trivy and dockle vulnerabilities

### PR DESCRIPTION
**Pull request by shigupt202**
1. Default severity threshold set to HIGH for trivy
2. Default level for dockle set to WARN
3. Removed dockle vulnerabilities with level IGNORE in logs

### CODE REVIEW

File: src/dockleHelper.ts [hunk 1]
```diff
@@ -134,14 +136,16 @@ export function printFormattedOutput() {
     let titles = [TITLE_VULNERABILITY_ID, TITLE_TITLE, TITLE_SEVERITY, TITLE_DESCRIPTION];
     rows.push(titles);
     dockleOutputJson[KEY_DETAILS].forEach(ele => {
-                let row = [];
-                row.push(ele[KEY_CODE]);
-                row.push(ele[KEY_TITLE]);
-                row.push(ele[KEY_LEVEL]);
-                row.push(ele[KEY_ALERTS][0]);
-                rows.push(row);           
+        if (ele[KEY_LEVEL] != LEVEL_IGNORE && ele[KEY_LEVEL] != LEVEL_INFO) {
```
Comment (thesattiraju) [inline src/dockleHelper.ts:139]: Why skip info?

**Review by thesattiraju — APPROVED** [review]

File: src/dockleHelper.ts (hunk 1 above)
Comment (shigupt202) [reply src/dockleHelper.ts:139]: Had a discussion with Vishal on this. He suggested to keep only WARN and FATAL.

File: action.yaml [hunk 2]
```diff
@@ -5,8 +5,9 @@ inputs:
     description: 'Docker image to scan'
     required: true
   severity-threshold:
-    description: 'severities of vulnerabilities to be displayed (UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL)'
+    description: 'Severities of vulnerabilities to be displayed (UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL)'
```
Comment (thesattiraju) [inline action.yaml]: `Minimum severity` or something on those lines perhaps

Comment (shigupt202) [reply action.yaml]: Okay

**Pull request merged**